content/_data/changelogs/lts.yml

```diff
@@ -4785,6 +4785,327 @@
+
+  changes: # compared to lts_baseline 2.277 - extracted from the RC commit(s)
+
+  - type: major bug
+    category: major bug
+    pull: 5282
+    issue: 41727
+    authors:
+    - timja
+    message: |-
+      Show available plugin updates by reloading update center data on upgrade/downgrade.
+
+  # Regressions introduced after 2.263.4 and resolved before 2.277.1 was released
+  #
+  # pull: 5216 (PR title: [JENKINS-64746] public API hudson.model.UpdateCenter.HudsonUpgradeJob constructor signature change)
+  # pull: 5236 (PR title: [JENKINS-64775] Restores plugin details in DOM)
+  # pull: 5263 (PR title: [JENKINS-64805] Fixes IE11 error on the available tab)
+  # pull: 5281 (PR title: [JENKINS-64840] Fix searching multiple update sites)
+  # pull: 5285 (PR title: Bump spring-security-bom from 5.4.2 to 5.4.4)
+
+  lts_changes: # compared to lts_predecessor 2.249.3 (selected by personal review)
+
+  - type: major rfe
+    category: major rfe
+    pull: 3895
+    issue: 56109
+    authors:
+    - jsoref
+    - timja
+    - fqueiruga
+    message: |-
+      Change Jenkins configuration UI from tables to divs for layout in forms.
+
+  - type: major rfe
+    category: major rfe
+    pull: 5051
+    issue: 64196
+    authors:
+    - timja
+    message: |-
+      Massive performance enhancement to available plugins page of Plugin Manager.
+      Exact matches of plugin name are moved to the top.
+
+  - type: bug
+    category: bug
+    pull: 5051
+    issue: 63684
+    authors:
+    - timja
+    message: |-
+      Fix incorrect striping of rows on available page of Plugin Manager.
+
+  - type: bug
+    category: bug
+    pull: 5145
+    issue: 64439
+    authors:
+    - benebsiny
+    message: |-
+      Prevent user input of 'e' or 'E' as 'positive-number', 'non-negative-number', or 'number'.
+
+  - type: bug
+    category: bug
+    pull: 5010
+    issue: 35452
+    authors:
+    - jeffret-b
+    message: |-
+      Change the standard URL for obtaining the inbound agent configuration file from <code>${agent_url}/slave-agent.jnlp</code> to <code>${agent_url}/jenkins-agent.jnlp</code>.
+      The old name is obsolete and will be removed at a future time.
+
+  - type: bug
+    category: bug
+    pull: 5043
+    authors:
+    - jeffret-b
+    references:
+      - pull: 5043
+      - url: "https://github.com/jenkinsci/remoting/releases/tag/remoting-4.6"
+        title: Remoting 4.6 changelog
+    message: |-
+      Upgrade from Remoting 4.5 to Remoting 4.6 with bugfixes and dependency updates.
+
+  - type: rfe
+    category: rfe
+    pull: 5035
+    issue: 64039
+    authors:
+    - jtnord
+    message: |-
+      Improve performance of authorisation strategies when the authentication realm is case insensitive.
+
+  - type: rfe
+    category: rfe
+    pull: 5102
+    authors:
+    - daniel-beck
+    message: |-
+      Stop bundling CVS plugin.
```

Comment:
I would probably move this down, it's less important than everything except the dependency updates I think